src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1389,10 +1306,18 @@ private BoundExpression GetDefaultParameterSpecialNoConversion(SyntaxNode syntax
             for (int i = 0; i < parameters.Length; i++)
             {
                 var parameter = parameters[i];
-                if (!visitedParameters[parameter.Ordinal] && parameter.IsOptional)
+                if (!visitedParameters[parameter.Ordinal])
                 {
+                    // Params array is filled in the local rewriter
```

Comment:
In the old LocalRewriter code, the params array would have been filled in before default parameters were looked at, so this wasn't necessary there.